src/test/llmnode_test.cpp

```diff
@@ -122,7 +131,7 @@ TEST_F(LLMFlowKfsTest, LLMNodeNameMissing) {
             output_stream: "RESPONSE:out2"
             node_options: {
                 [type.googleapis.com / mediapipe.LLMCalculatorOptions]: {
-                    workspace_path: "/workspace/"
+                    models_path: "/workspace/"
```

Comment:
later, also relative path should be supported...

Reply:
@rasapala I think when you added node initialization you take that into account. Can you confirm it should work?

Reply:
Yeah it should work, but I have not tested it yet. Maybe with the model tests we can add test for relative? Or next PR.